src/Sarif.Viewer.VisualStudio/ResultTextMarker.cs

```diff
@@ -128,116 +303,57 @@ public bool TryNavigateTo(bool usePreviewPane)
-            if (this.PersistentSpanValid())
+            if (!this.PersistentSpanValid())
             {
-                if (!SdkUIUtilities.TryGetActiveViewForTextBuffer(this.tag.DocumentPersistentSpan.Span.TextBuffer, out IWpfTextView wpfTextView))
-                {
-                    return false;
-                }
-
-                ITextSnapshot currentSnapshot = this.tag.DocumentPersistentSpan.Span.TextBuffer.CurrentSnapshot;
-
-                // Note that "GetSpan" is not really a great name. What is actually happening
-                // is the "Span" that "GetSpan" is called on is "mapped" onto the passed in
-                // text snapshot. In essence what this means is take the "persistent span"
-                // that we have and "replay" any edits that have occurred and return a new
-                // span. So, if the span is no longer relevant (lets say the text has been deleted)
-                // then you'll get back an empty span.
-                SnapshotSpan trackingSpanSnapshot = this.tag.DocumentPersistentSpan.Span.GetSpan(currentSnapshot);
-
-                // If the caret is already in the text within the marker, don't re-select it
-                // otherwise users cannot move the caret in the region.
-                // If the caret isn't in the marker, move it there.
-                if (!trackingSpanSnapshot.Contains(wpfTextView.Caret.Position.BufferPosition) &&
-                    !trackingSpanSnapshot.IsEmpty)
-                {
-                    wpfTextView.Selection.Select(trackingSpanSnapshot, isReversed: false);
-                    wpfTextView.Caret.MoveTo(trackingSpanSnapshot.End);
-                    wpfTextView.Caret.EnsureVisible();
-                    wpfTextView.VisualElement.Focus();
-                }
-
-                return true;
+                return false;
             }
 
-            return false;
-        }
-
-        /// <summary>
-        /// Select current tracking text with <paramref name="highlightColor"/>. 
-        /// If highlightColor is null than code will be selected with color from <seealso cref="Color"/>.
-        /// If the mark doesn't support tracking changes, then we simply ignore this condition (addresses VS crash 
-        /// reported in Bug 476347 : Code Analysis clicking error report C6244 causes VS2012 to crash).  
-        /// Tracking changes helps to ensure that we navigate to the right line even if edits to the file
-        /// have occurred, but even if that behavior doesn't work right, it is better to 
-        /// simply return here (before the fix this code threw an exception which terminated VS).
-        /// </summary>
-        /// <param name="highlightColor">Color</param>
-        public void AddTagHighlight(string highlightColor)
-        {
-            if (this.tag != null)
+            if (!SdkUIUtilities.TryGetActiveViewForTextBuffer(this.persistentSpan.Span.TextBuffer, out IWpfTextView wpfTextView))
```

Comment:
Comment that this is here because the document can be "open" but not in a visible tab. That makes the persistent span valid (hence all the code above is skipped) but the user cannot see the tab.